### ISSUE TITLE
EmptyLinesAroundAttributeAccessor should ignore `# rubocop` comments

### ISSUE THREAD

**Issue by JoeCohen** [rubocop/rubocop#10693]
## Is your feature request related to a problem? Please describe.

The following generates a Style/EmptyLinesAroundAttributeAccessor offense.
```ruby
    # Disable cop to allow comments on each accessor in a separate line
    # rubocop:disable Style/AccessorGrouping
    attr_accessor :var      # name of global variable to assign it to if any
    attr_accessor :points   # array of [lat, long]
    attr_accessor :color    # css-format color string
    attr_accessor :weight   # width of line in pixels
    attr_accessor :opacity  # from 0.0 to 1.0
    # rubocop:enable Style/AccessorGrouping

    more code
```

## Describe the solution you'd like

Rubocop should not generate an offense if the accessors are followed by a `# rubocop:` comment. 

## Describe alternatives you've considered
I considered excluding the file in the cop. But that would turn off the cop for the entire file. 
I don't like that because it means that Rubocop will not detect other accessors that could be grouped.

## Additional context
rubocop 1.30.0
